controllers/list.controller.js

```diff
@@ -1,11 +1,18 @@
 const { list } = require("../models");
 
-exports.get = async (req, res) => {
-  try {
-    const all = await list.findAll();
-    res.json(all);
+exports.remove = (req, res) => {
+  const { id } = req.params;
+  if (!id.trim()) {
+    res.status(422).send({answer: "Invalid id"})
   }
-  catch (err) {
+  list.destroy({where: {id}})
+  .then(async(removed) =>  {
+    if(removed) {
+      const all = await list.findAll();
```

Comment:
please use method that you wrote on your branch where you're getting information